docker-compose.yml

```diff
@@ -38,4 +46,21 @@ services:
       - api
     networks:
       - default
-    restart: always
\ No newline at end of file
+    logging:
+      driver: "syslog"
+      options:
+        syslog-address: "tcp://mozdef_logging_agent:514"
+    restart: always
+  mozdef_logging_agent:
+    build:
+      context: .
+      dockerfile: ./docker/mozdef_logging_agent
+    command: INSERT_COMMAND
+    environment:
+      - INSERT_SECRET_NAME_AND_DEFAULT_VALUE_EXAMPLE1
+      - INSERT_SECRET_NAME_AND_DEFAULT_VALUE_EXAMPLE2
+    ports:
+      - 514:514
```

Comment:
This shouldn't be needed for inter-docker container comms if these are linked.  However, if we want to also throw host-OS/App logs into the container and then MozDef, we'll need to expose this service on at least the localhost.

Reply:
....which reminds me, I need to link this to the other containers if I plan to make this work.